src/matlab2tikz.m

```diff
@@ -4005,17 +4010,37 @@ function warnFacetedInterp(m2t)
     else
         arrowLength = '{sqrt((\thisrow{u})^2+(\thisrow{v})^2)}';
     end
+
     plotOpts = opts_add(plotOpts, 'point meta', arrowLength);
+    plotOpts = opts_add(plotOpts, 'point meta min', '0');
 
     if showArrowHead
         %TODO: scale the arrows more rigorously to match MATLAB behavior
-        %There is a "MaxHeadSize" property (at least in R2014b) that plays a
-        %role in determining the quiver size.
+        %There is a "MaxHeadSize" property that plays a role.
+        % Currently, this is quite hard to do, since the size of the arrows
+        % is defined in pgfplots in absolute units (here we specify that those
+        % should be scaled up/down according to the data), while the data itself
+        % is in axis coordinates (or some scaled variant). I.e. we need the
+        % physical dimensions of the axis to compute the correct scaling!
+        userInfo(m2t, ['Please change the "arrowHeadSizeFactor" option', ...
```

Comment:
What code are you referring to? As I see it, the code `userInfo(...)` is already after the comments `% There is a ...`.